packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -13,8 +14,8 @@ const selectors = {
 	// Cart item
 	cartItem: ( itemName: string ) =>
 		`[data-testid="review-order-step--visible"] .checkout-line-item >> text=${ itemName.trim() }`,
-	removeCartItemButton: ( itemName: string ) =>
-		`[data-testid="review-order-step--visible"] button[aria-label*="Remove ${ itemName.trim() } from cart"]`,
+	removeCartItemButton: () =>
+		`[data-testid="review-order-step--visible"] button.checkout-button.checkout-line-item__remove-product`,
```

Comment:
Not sure why this change is included. It does not appear to be used anywhere in rest of the PR, and it is used in `CartCheckoutPage.removeCartItem` which is in turn used by numerous specs.